### DISCUSSION TITLE
copilot4eclipse chat window is often never responding with an answer

### DISCUSSION THREAD

**Discussion by davidmichaelkarr** [Genuitec/Copilot4Eclipse-community#49] [Help]
Multiple times today I have entered a question in the chat window, and it will print the file it's referencing, and it will provide a link for information for a related question, but never actually answer the question. It isn't 100% repeatable. I've seen it happen multiple times today, but other times it provided an answer. I saw it doing this, then I got an answer, and then it didn't respond again.

I'm using the latest available version.

**Comment by wayneparrott** [comment]
We will need more information to determine what might be the problem.
Please try the following next time you observe Chat in this state:

1) enable [low level logging](https://www.genuitec.com/products/copilot4eclipse/docs/basics/logging#logging-communications-with-github-copilot-services-low-level) *editied link*
2) submit a Chat prompt/request
3) When it completes with no info, disable logging and send us the log file for review. Send to support@genuitec.com and include a reference to this discussion.

Wayne
Copilot4Eclipse Team

**Reply by davidmichaelkarr** [reply]
That link points to localhost. I think I've set up logging before, but I can't find the notes on how to do it.

**Reply by davidmichaelkarr** [reply]
Never mind, I found the notes on how to enable it.

**Reply by wayneparrott** [reply]
my bad on that link. I corrected it. Here's the proper link as well: https://www.genuitec.com/products/copilot4eclipse/docs/basics/logging#logging-communications-with-github-copilot-services-low-level
